crud-s3/README.md

```diff
@@ -0,0 +1,36 @@
+# Python3 CRUD + AWS S3 Integration With Functions
+Create, Read, Update, and Delete entries in an S3 bucket.
+
+# What does it do?
+Creates a simple crud service using only binaris functions written in python3.
+
+# Deploy me!
+
+[Get a Binaris API Key](https://binaris.com/) and an aws account (free trial account available from [Amazon](https://aws.amazon.com/console/)).
+
+```bash
+# Install the Binaris CLI
+$ npm install -g binaris
+
+# Set your API key in ~/.binaris.yml
+$ bn login
+
+# Clone this repo.
+$ git clone git@github.com:binaris/functions-examples.git
+$ cd crud-s3
+
+# Pull the bob repo so you can compile the pip dependencies.
+$ git clone git@github.com:binaris/bob.git
+
+# Set the environment variables for deployment
```

Comment:
since you already deploy via a script, `deploy.sh`, why not add to that script `AWS_ACCESS_KEY=$(aws configure get aws_access_key_id)` and save the user some effort?

Reply:
👍 fair point, making change.